src/cpp/session/modules/build/SessionInstallRtools.cpp

```diff
@@ -89,6 +92,31 @@ Error installRtools()
    if (error)
       return error;
 
+   if (version == "4.2")
```

Comment:
I still need to test this out on Windows, but wanted to put up for review to see if this is even a good idea